src.ts/account.ts

```diff
@@ -107,7 +107,7 @@ export class Account {
         try {
             result = await this.connection.provider.sendTransaction(signedTx);
         } catch (error) {
-            if (error.message === '[-32000] Server error: send_tx_commit has timed out.') {
```

Comment:
looks like should have both checks for compatibility with testnet

Reply:
I moved this check to provider to generate TimeoutError there